crates/api/src/v1/package.rs

```diff
@@ -120,6 +120,9 @@ pub enum PackageError {
     /// The operation was not supported by the registry.
     #[error("the requested operation is not supported: {0}")]
     NotSupported(String),
+    /// The provided content was rejected by server policy.
+    #[error("the content was rejected by server policy: {0}")]
+    ContentPolicyRejection(String),
```

Comment:
I wonder if we need to make it specifically a `ContentPolicyRejection` instead of just a `Rejection` with reason. I think there are going to be several kinds of policies and I think in many cases e.g. "content is malformed" some manual effort is going to be needed anyway to solve the problem and we don't need to categorize them in detail.

Reply:
Agreed, I'll make this more generic.